bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java

```diff
@@ -135,11 +136,11 @@ private synchronized void poll() {
             logger.debug("Polling for state");
             pollShades();
             pollScenes();
-        } catch (IOException e) {
-            logger.debug("Could not connect to bridge", e);
+        } catch (ProcessingException e) {
+            logger.debug("Error connecting to bridge: {}", e.getMessage());
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, e.getMessage());
         } catch (Exception e) {
```

Comment:
Can you be more explicit here? Usually catching `Exception` is not the best decision.

Reply:
Ok. Done.